<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StakingPage</class>
 <widget class="QWidget" name="StakingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item row="1" column="0" rowspan="2" colspan="3">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>25</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>25</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>1</number>
            </property>
            <item row="4" column="0">
             <widget class="QLabel" name="label40Text">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>40 BAY</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label20Text">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>20 BAY</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label40Amount">
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Total of coins that was staked, and do not yet count toward the current balance</string>
              </property>
              <property name="text">
               <string notr="true">0 BAY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label10Text">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>10 BAY</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label20Count">
              <property name="text">
               <string>(0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="labelTotalAmount">
              <property name="text">
               <string>0 BAY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label10Count">
              <property name="text">
               <string>(0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label5Count">
              <property name="text">
               <string>(0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label5Amount">
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Your current spendable balance</string>
              </property>
              <property name="text">
               <string notr="true">0 BAY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label10Amount">
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Your current reserve balance</string>
              </property>
              <property name="text">
               <string>0 BAY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="labelRewards">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>STAKE INPUTS</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="labelTotalCount">
              <property name="text">
               <string>(0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>25</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label20Amount">
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Your current frozen balance</string>
              </property>
              <property name="text">
               <string>0 BAY</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="labelTotalText">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>TOTAL</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label40Count">
              <property name="text">
               <string>(0)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label5Text">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>5 BAY</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="3">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>25</number>
            </property>
            <property name="topMargin">
             <number>25</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelText">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QCheckBox" name="checkConsolidateCoins">
              <property name="text">
               <string>Consolidate coins on stake</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelConsolidateCoins">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>        (add separate transaction to consolidate coins per address)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkRewardTo">
              <property name="text">
               <string>Reward to address:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>7</number>
              </property>
              <item>
               <widget class="QValidatedLineEdit" name="lineRewardTo">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>The address to send the reward to. If empty, stake reward will remain attached to address that staked it.</string>
                </property>
                <property name="maxLength">
                 <number>34</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="rewardAddressBookButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Choose address from address book</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../bitbay.qrc">
                  <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
                </property>
                <property name="shortcut">
                 <string>Alt+A</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkSupportFund">
              <property name="text">
               <string>Support BITBAY marketing fund:</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>7</number>
              </property>
              <item>
               <widget class="QValidatedLineEdit" name="lineSupportTo">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="toolTip">
                 <string>The marketing fund address to send the reward to.</string>
                </property>
                <property name="maxLength">
                 <number>34</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="supportAddressBookButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Choose address from address book</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../bitbay.qrc">
                  <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
                </property>
                <property name="shortcut">
                 <string>Alt+A</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QSlider" name="sliderSupport">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSupportValue">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelSupportPercent">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkConsolidateCoins</tabstop>
  <tabstop>checkRewardTo</tabstop>
  <tabstop>lineRewardTo</tabstop>
  <tabstop>rewardAddressBookButton</tabstop>
  <tabstop>checkSupportFund</tabstop>
  <tabstop>lineSupportTo</tabstop>
  <tabstop>supportAddressBookButton</tabstop>
  <tabstop>sliderSupport</tabstop>
 </tabstops>
 <resources>
  <include location="../bitbay.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkRewardTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineRewardTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkRewardTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>rewardAddressBookButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSupportFund</sender>
   <signal>toggled(bool)</signal>
   <receiver>sliderSupport</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSupport</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelSupportValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSupportFund</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSupportTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSupportFund</sender>
   <signal>toggled(bool)</signal>
   <receiver>supportAddressBookButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
